### PR TITLE
Fix some broken links and bad markdown syntax

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

This PR cleans up some broken links and bad markdown syntax

#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/flow/tutorials/flow-by-example/get-started.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ aliases:
 - /docs/grafana-cloud/monitor-infrastructure/agent/flow/tutorials/flow-by-example/faq/
 - /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/tutorials/flow-by-example/faq/
 - /docs/grafana-cloud/send-data/agent/flow/tutorials/flow-by-example/faq/
-canonical: https://grafana.com/docs/agent/latest/flow/tutorials/flow-by-example/faq/
+canonical: https://grafana.com/docs/agent/latest/flow/tutorials/flow-by-example/
```
Comment (jdbaldry) [inline docs/sources/flow/tutorials/flow-by-example/get-started.md:7]:
```suggestion
canonical: https://grafana.com/docs/agent/latest/flow/tutorials/flow-by-example/get-started/
```

Comment (clayton-cornell) [reply docs/sources/flow/tutorials/flow-by-example/get-started.md:7]: D'oh.....

**Review by jdbaldry — APPROVED** [review]
Changes look good to me.

While checking locally, I spotted that https://github.com/grafana/agent/pull/6596/files#diff-2dd8b4ff97df653497acbd02afbb3afd74d0bbdac2abd7767c6be1df1f4808a5 doesn't have an H1 heading but I can't supply a suggestion because it's outside of the PR diff context.

**Comment by clayton-cornell** [comment]
I just saw that missing H1 too (it's easy to miss)... I'll get it cleaned up in this PR.

**Review by rfratto — APPROVED** [review]

**Pull request merged**